references/detection/transforms.py

```diff
@@ -591,3 +591,104 @@ def forward(
     def __repr__(self) -> str:
         s = f"{self.__class__.__name__}(blending={self.blending}, resize_interpolation={self.resize_interpolation})"
         return s
+
+
+class Mosaic(nn.Module):
+    """
+    Mosaic Transform
+    """
+
+    def __init__(self, min_frac: float = 0.25, max_frac: float = 0.75, size_limit: int = 10) -> None:
+        super().__init__()
+        self.min_frac = min_frac
+        self.max_frac = max_frac
```

Comment:
here we need to check if the min_frac and max_frac arguments are in between 0 and 1.